### DISCUSSION TITLE
Action "Add to Safari reading list"

### DISCUSSION THREAD

**Discussion by bobbyfuller** [ViennaRSS/vienna-rss#1419] [Q&A] [answered]
Hi,
having installed the version 3.7.0, I found out, that it's not possible to add articles to the Safari reading list.
Which made me downgrade to the previous version. 
Is there a reason, why this useful feature was removed?
Greetings
Alexander

**Comment by Eitot** [comment]
Can you please elaborate:
- Where does the problem occur?
- Which older version the problem does not appear?

**Reply by bobbyfuller** [reply]
In version 3.6.2 it was possible to trigger this action, either via menu "Articles" or using an icon in the toolbar.
It's not available anymore in version 3.7.0
![Bildschirmfoto 2021-01-02 um 20 32 25](https://user-images.githubusercontent.com/76658073/103465201-baf49d80-4d39-11eb-8f53-96174bc30b4c.jpg)

**Comment by Eitot** [comment]
Can you check the Vienna.app bundle and see if it has empty folders in Contents → SharedSupport → Plugins? If so, can you report back?

The issue might be caused by the duplicate/empty folder names. It seems that the issue pertains to the copy of Vienna hosted at Bintray (which is also the source of Sparkle updates). Downloading a copy from SourceForge (see here: https://github.com/ViennaRSS/vienna-rss/releases/latest) is a workaround.

**Reply by bobbyfuller** [reply]
Yes, assumption is right. Folders under Contents → SharedSupport → Plugins all are empty. 
When I download the application from Sourceforge, it looks the same to me. Subdirectories are also empty in that version.

However, I was able to reactivate the action "Add to Safari reading list" by copying the respective files from the old version. 
So I am back on version 3.7.0 including this functionality.

Thanks a lot for the quick reply.

**Comment by josh64x2** [comment, marked as answer]
Fixed in 3.7.1
